test/configs/test-manifest-debugging.xml

```diff
@@ -0,0 +1,111 @@
+<?xml version="1.0" encoding="utf-8"?>
+<OfficeApp xmlns="http://schemas.microsoft.com/office/appforoffice/1.1"
+  xmlns:xsi="http://www.w3.org/2001/XMLSchema-instance"
```

Comment:
Why a different manifest for debugging?

Reply:
The same reason as the other webpack config, just to change port from 3000 to 3001